src/Helpers/ConfigureEnvironmentServiceProvider.php

```diff
@@ -32,7 +34,7 @@ public function register(): void
     protected function setTimeZone(CarbonTimeZone $timeZone): void
     {
         ini_set('date.timezone', (string)$timeZone);
-        date_default_timezone_set($timeZone);
+        date_default_timezone_set((string)$timeZone);
```

Comment:
space between casting and variable

Reply:
I cannot see a space here…

Reply:
Oh. I think I got your comment wrong. You mean you want to have a space there. Hard to guess from half a sentence…

Reply:
https://github.com/engelsystem/engelsystem/pull/1056